.github/ISSUE_TEMPLATE/replay_issue.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Replay Issue
+about: Can not replay a web specific web page? Tell us about it!
+
+---
+## Page
+<!--- Tell us about the page, by at least giving us a URL! -->
+<!--- Here is the perfect place to include any details about how we failed to replay the page. -->
```

Comment:
Maybe change to 'Here is the perfect place to include any details about what is going wrong with the page compared to what is expected.

Reply:
How about the classic "expected behavious" and "what actually happened"?

Reply:
Yes the classics are always better so per @despens comment will use "expected" vs "actual" here. 
Do we wish to keep the steps to reproduce number list example or just leave it blank?


### Changes to be applied
**Expected behavior**
`<!--- Tell us about the page and how it should replay -->`


**What actually happened**
`<!--- Tell us about how we failed to replay the page -->`